### PR TITLE
Added new delegate Ben Allen (BEN)

### PR DESCRIPTION
**Pull request by ben-allen**

### CODE REVIEW

File: delegates.txt [hunk 1]
```diff
@@ -41,6 +41,7 @@ Augustus Yuan (AYN)
 Avik Chaudhuri (AVC)
 Axel Rauschmayer (ARR)
 Aysegul Yonet (AYS)
+Ben Allen (BEN)
```
Comment (ljharb) [inline delegates.txt]: our typical convention would have this be `BAN`, is that ok?

Comment (ben-allen) [reply delegates.txt]: Shoot, I knew snagging BEN was too good to be true. BAN is fine, though if possible BJA would be preferable

Comment (ljharb) [reply delegates.txt]: Also, where's the matching reflector issue for this addition? Can you have your Ecma member company point of contact file one?

Comment (ryzokuken) [reply delegates.txt]: @ljharb apologies, was waiting on the PR link before posting.

Comment (robpalme) [reply delegates.txt]: Nudging this forwards...

```suggestion
Ben Allen (BAN)
```

**Review by ljharb — APPROVED** [review]

**Pull request merged**